core/src/test/java/org/axonframework/commandhandling/model/SomeAnnotatedFactoryMethodClass.java

```diff
@@ -0,0 +1,27 @@
+package org.axonframework.commandhandling.model;
+
+import org.axonframework.commandhandling.CommandHandler;
+
+/**
+ * Test Aggregate demonstrating the use of a factory method for @{@link Aggregate} instantiation.
+ *
+ * @author Christophe Bouhier
+ */
+public class SomeAnnotatedFactoryMethodClass {
```

Comment:
Could this class be an inner class to the test that's using it? Makes it more readable, IMO.